### DISCUSSION TITLE
Lifetimes Model Application & interpretation

### DISCUSSION THREAD

**Discussion by KollyT** [pymc-labs/pymc-marketing#1401] [Q&A]
Hello,

I am currently using the lifetimes package to try and determine which of our consumers are currently 'Alive' or 'Dead' based on their transactional information. I have successfully applied the model but wondered if you could give some guidance on how to interpret the outputs. I have two key questions:

- When you have consumers that are deemed 95% alive versus others at 30% alive, is this score based on the individual consumer and their change in behaviour or is it in relation to the total population?
 
- If I wanted to identify lapsing consumers faster would a drop in the probability alive score from one period to the next be the best method? or just using their current prob alive score, i.e. anyone under 30% alive will be considered lapsing/lapsed?

I would really appreciate any help you can provide on the above.

**Comment by ColtAllen** [comment]
Hey @KollyT,

Have you created an equivalent model yet with the `pymc-marketing` library?

> * If I wanted to identify lapsing consumers faster would a drop in the probability alive score from one period to the next be the best method? or just using their current prob alive score, i.e. anyone under 30% alive will be considered lapsing/lapsed?

Which model are you using? The `ParetoNBDModel` would be better suited for this. You're correct that less than 30% is a good rule of thumb for an at-risk/lapsed customer.

> * When you have consumers that are deemed 95% alive versus others at 30% alive, is this score based on the individual consumer and their change in behaviour or is it in relation to the total population?

Technically, the score is based on both. The model is fit to overall population data, but predictions are conditioned on an individual customer's _recency, frequency,_ and _T_.

**Comment by KollyT** [comment]
Thank you Colt! 

I haven't switched over to the PYMC-marketing library yet, but planning to in future iterations of the model. I am using the ParetoNBD model, glad it is the right approach. 

In regards to identifying lapsing consumers, can a drop in ProbabilityAlive score from one period to another i.e. Jan=98% and Feb=95.8% also indicate lapsing? Would this be an accurate method of idenifying consumers that are dropping off?

Thank you!

**Comment by ColtAllen** [comment]
> In regards to identifying lapsing consumers, can a drop in ProbabilityAlive score from one period to another i.e. Jan=98% and Feb=95.8% also indicate lapsing? Would this be an accurate method of idenifying consumers that are dropping off?

It can, but it's important to note that ProbabilityAlive estimates in future time periods assume no purchases were made. This notebook provides more information:

https://www.pymc-marketing.io/en/stable/notebooks/clv/pareto_nbd.html

**Comment by ColtAllen** [comment]
@KollyT can I close this?
